libs/s3/s3_multipart.py

```diff
@@ -317,10 +276,7 @@ def multipart_download(self, **kwargs):
         assert key
         try:
             self.s3_resource.Object(bucket_name, key). \
-                download_file(file_path,
-                              Config=config,
-                              Callback=ProgressPercentage(file_path)
-                              )
+                download_file(file_path, Config=config, Callback=ProgressPercentage(file_path))
         except Exception as error:
-            LOGGER.error("Error in multipart_upload: %s", str(error))
+            LOGGER.exception("Error in multipart_upload: %s", str(error))
```

Comment:
Let's update this to "Error in multipart_download" and while we're at it, we can also use the newly introduced constant - ERR_MSG